packages/flame/lib/src/components/base_component.dart

```diff
@@ -171,6 +173,13 @@ abstract class BaseComponent extends Component {
     }
   }
 
+  Future<void> addChildren(
```

Comment:
This method needs to be async? You don't use any async operations on the method body

Reply:
Ah, forgot that the user should be able to wait for the children to be loaded. Does it look correct now?

Reply:
Did you changed anything? Can't spot any difference, there is still nothing being awaited on the  function body

Reply:
Maybe I commented before I had pushed, it is there now at least.
```dart
    await Future.wait(
      children.map(
        (child) => addChild(child, gameRef: gameRef),
      ),
    );
```

Reply:
👍